compute_tools/src/spec_apply.rs

```diff
@@ -444,7 +444,7 @@ async fn get_operations<'a>(
         }
         ApplySpecPhase::RunInEachDatabase { db, subphase } => {
             match subphase {
-                PerDatabasePhase::DropSubscriptionsForDeletedDatabases => {
+                PerDatabasePhase::DropLogicalSubscriptions => {
```

Comment:
SQL file should be probably renamed as well `drop_subscription_for_drop_dbs.sql` -> `drop_subscriptions.sql`?